Documentation update - Closes #699 (#701)
LiskHQ/lisk-service: docs/config_options.md

```bash
## Lisk Service Core

# Local Redis cache with persistency for Core microservice
# Refer to the /docker/redis/redis.persistent.conf for more details
# Note: SERVICE_BROKER uses a different DB
SERVICE_CORE_REDIS=redis://localhost:6379/1

# Local Redis LRU cache for Core microservice

> # This should be a different instance on production

Reviewer: ```suggestion
# This should be a separate instance in the production
```
